src/main/java/org/example/homewirk_nr_8/Animal.java

```diff
@@ -0,0 +1,15 @@
+package org.example.homewirk_nr_8;
+
+public interface Animal {
+    String name = null;
```

Comment:
The variables that you declare in an interface are final by their nature. Also, because of that variables that are declared in an interface should have the following naming convention: INTERFACE_VARIABLE_EXAMPLE. I don't see where you use these variables. Either remove them or assign a value that is going to be used in the future